internal/cmd/root.go

```diff
@@ -34,9 +36,34 @@ func init() {
 
 	rootCmd.PersistentPostRun = func(cmd *cobra.Command, args []string) {
 		settings.PersistChanges()
+		if version == "dev" {
+			return
+		}
+		settings, err := settings.ReadSettings()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to read settings: %s\n", err)
```

Comment:
This is a background op that is not related to what the user wanted to do.
WDYT about failing silently?